extensions/azurecore/package.nls.json

```diff
@@ -19,5 +19,5 @@
 	"config.enableUsGovCloudDescription": "Should US Government Azure cloud (Fairfax) integration be enabled",
 	"config.enableChinaCloudDescription": "Should Azure China integration be enabled",
 	"config.enableGermanyCloudDescription": "Should Azure Germany integration be enabled",
-	"config.useNewSignInExperience": "Use the new Azure sign in experience"
+	"config.enableDeviceCodeLogin": "Use device code login mechanism"
```

Comment:
you might add Azure into the string somewhere to provide additional context.  Maybe "Enable Azure Active Directory device code authentication"?